packages/KTable/KTable.vue

```diff
@@ -184,6 +216,19 @@ export default {
 
 table.k-table {
   border-collapse: collapse;
+
+  &.side-border {
+    border-collapse: separate;
+    border-spacing: 0 5px;
+
+    tr:hover {
```

Comment:
Idk that this should be a different color than the default hover value. Let’s double check with design but my thought is just use the predefined hover color and we can change it if it’s different per app

Reply:
the predefined hover color doesn't exist in KTable natively though